### PR TITLE
:seedling: Update readme for onboarding

### PR DESCRIPTION
**Pull request by ibolton336**
- This change updates the readme.md in preparation for on-boarding new developers to the project. The current readme was a little hard to follow and jumped around a bit. The intention here is to give all of the commands required to get our preferred (minikube) environment up and running in a sequential order. Additional info is added via links within the markdown & a troubleshooting section is added at the end of the doc to add any supplemental information.

### CODE REVIEW

**Review by gildub — APPROVED** [review]
LGTM

**Comment by ibolton336** [comment]
#565
